src/pages/Profile/Profile.css

```diff
@@ -13,6 +14,52 @@
          This will help you when you get started.
     */
 
+.pic{
+    background-color: rgb(191, 191, 191);
+    
+    width: 300px;
+    height: 300px;
+    border-radius: 50%;
+    margin-left: 200px;
+}
+
+.info{
+    display: flex;
+    flex-direction: column;
+}
+
+.name{
+    font-size: 40px;
+    margin-bottom: 20px;
+    margin-left: -10px;
+}
+
+.stats {
+    display: flex;
+    flex-direction: columnn; 
+    
+}
+
+
+.nums{
+    font-size: 40px;
+    display: flex;
+    justify-content: space-between;
+    width: 510px;
+    
+}
+
+.desc{
+    font-size: 20px;
+    display: flex;
+    justify-content: space-between;
+    width: 590px;
```

Comment:
We want to avoid setting a specific width for an element because that makes it static on that specific page.